source/GameInterface/Services/Armies/Extensions/ArmyExtensions.cs

```diff
@@ -25,13 +25,13 @@ internal static class ArmyExtensions
             .GetMethod("ApplyInternal", BindingFlags.NonPublic | BindingFlags.Static)
             .BuildDelegate<Action<Army, Army.ArmyDispersionReason>>();
 
-    internal static void AddPartyInternal(this MobileParty mobileParty, Army army)
+    internal static void AddPartyInternal(MobileParty mobileParty, Army army)
     {
         Army_OnAddPartyInternal(army,mobileParty);
     }
-    internal static void RemovePartyInternal(this MobileParty mobileParty, Army army)
+    internal static void RemovePartyInternal(MobileParty mobileParty, Army army)
```

Comment:
make this into an army extension
```suggestion
    internal static void RemovePartyInternal(this Army army, MobileParty mobileParty)
```